pioreactor/background_jobs/temperature_control.py

```diff
@@ -308,23 +311,27 @@ def evaluate_and_publish_temperature(self):
             previous_heater_dc = self.heater_duty_cycle
             self._update_heater(0)
 
-            N_sample_points = 17
+            N_sample_points = 15
```

Comment:
this is great - a modest decrease there doesn't effect the inference much, and allows us to hit a target temperature faster.